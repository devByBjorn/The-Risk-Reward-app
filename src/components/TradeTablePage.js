import React from 'react'
import ClosedTradeTable from './ClosedTradeTable'
import ActiveTradeTable from './ActiveTradeTable'
import PendingTradeTable from './PendingTradeTabel'
import SetDateRange from './TradeTableDatePick'
import AppTable from '../implementation_ideas/tables/tableRender'
import TableClosedTrades from '../implementation_ideas/tableMUI/TableClosedTrades'
import TableActiveTrades from '../implementation_ideas/tableMUI/TableActiveTrades'
import TablePendingTrades from '../implementation_ideas/tableMUI/TablePendingTrades'
import TableTabs from '../implementation_ideas/tableMUI/TableTabs'
import PageMainContainer from '../components_style/PageMainContainer'

const TradeTablePage = () => (
  <PageMainContainer>
    <TableTabs />
  </PageMainContainer>
)

export default TradeTablePage