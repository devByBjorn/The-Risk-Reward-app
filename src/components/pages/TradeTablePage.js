import React from 'react'
import { Link } from 'react-router-dom'
import ClosedTradeTable from '../tables/ClosedTradeTable'
import ActiveTradeTable from '../tables/ActiveTradeTable'
import PendingTradeTable from '../tables/PendingTradeTabel'
import TradeTableStats from '../table-complements/TradeTableStats'
import SetDateRange from '../table-complements/TradeTableDatePick'

const TradeTablePage = () => (
  <div>
    <h1>Trade Table Page</h1>
    <Link to="/add-trade">Add Trade</Link>
    <SetDateRange />
    <h3>Closed trades</h3>
    <ClosedTradeTable />
    <br />
    <h3>Stats closed trades</h3>
    {/*<TradeTableStats />*/}
    <br />
    <h3>Active trades</h3>
    <ActiveTradeTable />
    <br />
    <h3>Pending trades</h3>
    <PendingTradeTable />
    <br />
  </div>
)

export default TradeTablePage