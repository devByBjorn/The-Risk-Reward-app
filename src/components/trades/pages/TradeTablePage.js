import React from 'react'
import { Link } from 'react-router-dom'
import ClosedTradeTable from '../tables/closed-trades/ClosedTradeTable'
import ActiveTradeTable from '../tables/active-trades/ActiveTradeTable'
import PendingTradeTable from '../tables/pending-trades/PendingTradeTabel'
import SetDateRange from '../table-complements/TradeTableDatePick'

const TradeTablePage = () => (
  <React.Fragment>
    <h1>Trade Table Page</h1>
    <Link to="/add-trade">Add Trade</Link>
    <SetDateRange />
    <h3>Closed trades</h3>
    <ClosedTradeTable />
    <br />
    <h3>Active trades</h3>
    <ActiveTradeTable />
    <br />
    <h3>Pending trades</h3>
    <PendingTradeTable />
    <br />
  </React.Fragment>
)

export default TradeTablePage