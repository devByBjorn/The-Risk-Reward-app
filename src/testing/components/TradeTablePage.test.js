import React from 'react'
import { shallow } from 'enzyme'
import TradeTablePage from '../../components/TradeTablePage'

test('should render TradeTablePage', () => {
  const wrapper = shallow(<TradeTablePage />)
  expect(wrapper).toMatchSnapshot()
})