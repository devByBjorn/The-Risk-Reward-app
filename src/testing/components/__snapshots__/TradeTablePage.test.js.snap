// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TradeTablePage 1`] = `
<PageMainContainer
  mobileMargin="30% 1.5rem"
>
  <TableTabs />
</PageMainContainer>
`;
